pkg/apis/lbaas/v1/e2e_connection_test.go

```diff
@@ -40,7 +40,7 @@ func unavailableServerConnectionCheck(url string) {
 			resp, err := http.Get(url)
 			g.Expect(err).NotTo(HaveOccurred())
 			g.Expect(resp.StatusCode).To(Equal(http.StatusServiceUnavailable))
-		}, 5*time.Second, 1*time.Second).Should(Succeed())
+		}, 60*time.Second, 5*time.Second).Should(Succeed())
```

Comment:
This is not part of the ticket and should be moved into a new PR.

Reply:
This is part of this PR as it only affects/fixes integration tests.